### ISSUE TITLE
Blocked users should still show in followers/following lists

### ISSUE THREAD

**Issue by heikadog** [bluesky-social/social-app#1362]
**Is your feature request related to a problem? Please describe.**

I run a mute list and it'd be a handy QoL feature to see people I have blocked/that have blocked me in someone's following list, the same way mute lists show them and show labels like "Blocking You" etc. I don't want to go on an external website or unblock people temporarily to see if someone's following or being followed by a person; I should just be able to see it.

Currently, it's invisible.

**Describe alternatives you've considered**

- Not showing the accounts (as-is)

**Additional context**

I like how mute lists show labels so you can see certain attributes at a glance.

**Comment by mschwendener** [comment]
> Blocked users should still show in followers/following lists 

I don't agree. 

When I block somebody I don't want to be reminded of their account in any context, unless I look at the list of accounts that I have blocked. 

But I could imagine a setting, for example as a checkmark, to "include / show also blocked accounts". Default should be unchecked. 

---

From another issue thread I know that when I block somebody, I can't make them unfollow me. 

I understand that it is technically not possible, but it's confusing people 
https://www.reddit.com/r/BlueskySocial/comments/15u8v4t/comment/jyk52fc/?utm_source=reddit&utm_medium=web2x&context=3 (scroll down to the post from 2 days ago)

**Comment by heikadog** [comment]
They show on Twitter even if you have them blocked. I think it'd be useful in the case of like, a mute list, where you wanna see possible connections or something similar. It'd be helpful to see people who follow each other even if you've blocked them. It used to be that when you blocked people in a mute list on Bluesky it'd just disappear and that's no longer the case. I've found it helpful to fully limit people from interacting with me while also having them on a mute list.
